Examples/HelloWorld/HelloWorld.swift

```diff
@@ -17,7 +17,21 @@ extension HelloWorld: SceneDelegate {
              options: Scene.ConnectionOptions) {
     guard let windowScene = scene as? WindowScene else { return }
 
-    self.window = Window(windowScene: windowScene)
-    self.window?.makeKeyAndVisible()
+    let window = Window(windowScene: windowScene)
+    self.window = window
+
+    // The rootViewController allows to set a title and add controls
+    window.rootViewController = ViewController()
+
+    // Add a title
+    window.rootViewController?.title = "swift-win32 Hello World"
```

Comment:
Would you mind using "Swift/Win32 Hello World" instead?
```suggestion
    window.rootViewController?.title = "Swift/Win32 Hello World"
```